### PR TITLE
AMS-159: Add TWA scalability guide

### PR DESCRIPTION
**Pull request by willy-ahva**

### CODE REVIEW

File: reference/teamwork_assistant/scalability_guide.rst [hunk 1]
```diff
@@ -14,3 +14,30 @@ etc.) and technically (how to use the recalculate command) in a healthy way.
 
     To know more about the project completeness visit the `project completeness`_ reference.
 
+Compute Project Completeness
```
Comment (MarieMinasyan) [inline reference/teamwork_assistant/scalability_guide.rst]: _Computing_ :)

File: reference/teamwork_assistant/scalability_guide.rst [hunk 2]
```diff
@@ -14,3 +14,30 @@ etc.) and technically (how to use the recalculate command) in a healthy way.
...
+----------------------------
+
+.. _medium catalog: ../scalability_guide/representative_catalogs.html
+
+Before beginning to work with the Teamwork Assistant, you have to be aware that this feature has to compute a lot of
```
Comment (phaseinducer) [inline reference/teamwork_assistant/scalability_guide.rst]: "You must be aware"

File: reference/teamwork_assistant/scalability_guide.rst [hunk 3]
```diff
@@ -14,3 +14,30 @@ etc.) and technically (how to use the recalculate command) in a healthy way.
...
+data. The more products and attribute groups you have in a project, the more project completeness computing will take.
```
Comment (MarieMinasyan) [inline reference/teamwork_assistant/scalability_guide.rst]: _the more time project completeness computing will take_

Comment (phaseinducer) [inline reference/teamwork_assistant/scalability_guide.rst]: "the more time project completeness computing will take" ?

File: reference/teamwork_assistant/scalability_guide.rst [hunk 4]
```diff
@@ -14,3 +14,30 @@ etc.) and technically (how to use the recalculate command) in a healthy way.
...
+We realized some benchmarks with our `medium catalog`_ on MySQL to give you an idea of the time it can take. A medium
```
Comment (MarieMinasyan) [inline reference/teamwork_assistant/scalability_guide.rst]: _did_ instead of _realized_

File: reference/teamwork_assistant/scalability_guide.rst [hunk 5]
```diff
@@ -14,3 +14,30 @@ etc.) and technically (how to use the recalculate command) in a healthy way.
...
+catalog is about 3,66 millions of product values.
+
+
+In this benchmark the catalog contained 5 users with read/edit permissions on all 15 attribute groups and 5 users had
```
Comment (phaseinducer) [inline reference/teamwork_assistant/scalability_guide.rst]: "and 5 users with read/edit permissions"

**Pull request merged**